Library/Formula/twoping.rb

```diff
@@ -0,0 +1,18 @@
+class Twoping < Formula
+  desc "Ping utility to determine directional packet loss"
+  homepage "http://www.finnie.org/software/2ping/"
+  url "http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz"
+  sha256 "d6997cd1680151e6f7d5e60137d45cd41bf385d26029878afdaaf5dc4f63dcc4"
+
+  depends_on "Python"
```

Comment:
Correct, it'll work out of the box with system OS X Python 2.7 (or any 2.6/2.7).  Looks like Leopard was 2.5, Snow Leopard was 2.6, so should I replace that with:

```
depends_on :python if MacOS.version <= :leopard
```

(Modification of example from Python-for-Formula-Authors.md which suggests "<= :snow_leopard" for 2.7-only programs.)